Assets/Scripts/Weapons/ProjectileGun/Bullet.cs

```diff
@@ -21,9 +21,9 @@ private void OnTriggerEnter2D(Collider2D collide)
     {
         if (collide.gameObject.tag != "Bullet")
         {
-            EnemyHealth enemyHealth = collide.gameObject.GetComponent<EnemyHealth>();
-            enemyHealth.Damage(Damage);
-            Destroy(collide.gameObject);
+            NonPlayerHealth nonPlayerHealth = collide.gameObject.GetComponent<NonPlayerHealth>();
+            nonPlayerHealth.Damage(Damage);
+            // Destroy(collide.gameObject);

```

Comment:
Does this need to be commented out still?

Reply:
It doesn't need to exist anymore. Also, the bullet needs to be destroyed at the end of the collide. @Dezerack